File: app/src/main/java/com/rockon999/android/leanbacklauncher/MediaButtonBroadcastReceiver.java
package com.rockon999.android.leanbacklauncher;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class MediaButtonBroadcastReceiver extends BroadcastReceiver {
    public void onReceive(Context context, Intent intent) {
        Log.i("LeanbackLauncher", "Received media button event in LeanbackLauncher, ignoring.");
        abortBroadcast();
    }
}

File: app/src/main/java/com/amazon/tv/leanbacklauncher/MediaButtonBroadcastReceiver.kt
package com.amazon.tv.leanbacklauncher

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class MediaButtonBroadcastReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.i("LeanbackLauncher", "Received media button event in LeanbackLauncher, ignoring.")
        abortBroadcast()
    }

}